### DISCUSSION TITLE
Atlantis City

### DISCUSSION THREAD

**Discussion by Impactdustructi** [Povstalec/StargateJourney#33] [General]
Is Atlantis added yet or is it still a WIP in lantea

**Comment by Povstalec** [comment]
There is no Atlantis on Lantea as of now, there's only a placeholder Lantean Outpost.

I don't wanna make any promises on the time frame of when it could be added, since situations often change and who knows how the future will go. But given how huge of a project building the Atlantis would be and how much time I have for modding currently, don't expect it to appear in the mod for at least another year.
